ci-operator/step-registry/openstack/test/e2e/openstack-test-e2e-chain.yaml

```diff
@@ -1,6 +1,8 @@
 chain:
   as: openstack-test-e2e
   steps:
+  - ref: openstack-test-e2e-images
+  - ref: mirror-images-qe-test-images
```

Comment:
This broke all openstack tests: https://search.dptools.openshift.org/?search=mirror-images-qe-test-images&maxAge=48h&context=1&type=bug%2Bissue%2Bjunit&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job